pkg/cluster/driver/byo/byo.go

```diff
@@ -182,10 +182,10 @@ func (bd *ByoDriver) clusterInit() ([]byte, error) {
 	}
 	pkiInfo, err := kubepki.GeneratePKI(certOptions,
 		kubepki.KubeconfigRequest{
-			Path:          bd.KubeconfigPath,
-			Host:          bd.getKubeAPIServerIP(),
-			Port:          uint16(6443),
-			ServiceSubnet: bd.Config.ServiceSubnet,
+			Path:           bd.KubeconfigPath,
+			Host:           bd.getKubeAPIServerIP(),
+			Port:           uint16(6443),
+			ServiceSubnets: strings.Split(bd.Config.ServiceSubnet, ","),
```

Comment:
do the segments need to be trimmed

Reply:
They actually have to be comma separated with no whitespace for Kubernetes to accept the subnets.  It's worth validating that.  I've filed #295 to improve configuration validation.